<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacterEditor</class>
 <widget class="QWidget" name="CharacterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="mainTab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="setupTab">
      <attribute name="title">
       <string>Setup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="10,1">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,10,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="rigLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Rig:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="rigComboBox">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="currentText">
                <string>(none)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="dummyRigButton">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create Single Bone Dummy Rig&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create Dummy Single Bone Rig&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="rigImportButton">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Import Rig&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,0,0">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <widget class="QLabel" name="behaviorLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Main Behavior:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="behavioComboBox">
               <property name="editable">
                <bool>true</bool>
               </property>
               <property name="currentText">
                <string>(none)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="behaviorNewButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="behaviorImportButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QLineEdit" name="rigBoneCountLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="rigBoneCount">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Skeleton Bones:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="ragdollBoneCount">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Ragdoll Bones:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="ragdollBoneCountLineEdit">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="setupMirrrorinGroupBox">
         <property name="title">
          <string>Mirroring Properties:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Axis:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="mirroringXDoubleSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="mirroringYDoubleSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Z:</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="mirroringZDoubleSpinBox">
                <property name="decimals">
                 <number>5</number>
                </property>
                <property name="minimum">
                 <double>-1.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="mirroringComputePairingsButton">
                <property name="text">
                 <string>Automatically Compute Bone Pairings</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Select Bone:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTreeView" name="selectBoneTreeView"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Select Paired Bone:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTreeView" name="selectPairedBoneTreeView"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="animationSetsTab">
      <property name="toolTip">
       <string>Manage character's animation sets</string>
      </property>
      <attribute name="title">
       <string>Animation Sets</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,2,0,0">
             <item>
              <widget class="QLabel" name="setCurrentLabel">
               <property name="text">
                <string>Current Set:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="setCurrentComboBox"/>
             </item>
             <item>
              <widget class="QToolButton" name="setAddButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="setRemoveButton">
               <property name="text">
                <string>—</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="setTriggeringEvents">
                   <property name="text">
                    <string>Triggering Events:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="setEventAddButton">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="setEventRemoveButton">
                   <property name="text">
                    <string>—</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTableWidget" name="setTriggetingEventsView">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="showDropIndicator" stdset="0">
                  <bool>false</bool>
                 </property>
                 <property name="dragDropOverwriteMode">
                  <bool>false</bool>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <property name="showGrid">
                  <bool>false</bool>
                 </property>
                 <property name="rowCount">
                  <number>0</number>
                 </property>
                 <property name="columnCount">
                  <number>1</number>
                 </property>
                 <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="horizontalHeaderDefaultSectionSize">
                  <number>200</number>
                 </attribute>
                 <attribute name="horizontalHeaderHighlightSections">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="horizontalHeaderStretchLastSection">
                  <bool>true</bool>
                 </attribute>
                 <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                 </attribute>
                 <attribute name="verticalHeaderMinimumSectionSize">
                  <number>20</number>
                 </attribute>
                 <attribute name="verticalHeaderDefaultSectionSize">
                  <number>20</number>
                 </attribute>
                 <column/>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="setAttacksLabel">
                   <property name="text">
                    <string>Attacks:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="setAttacksAddButton">
                   <property name="text">
                    <string>+</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QToolButton" name="setAttacksRemoveButton">
                   <property name="text">
                    <string>—</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTreeWidget" name="setAttacksTreeWidget">
                 <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <attribute name="headerVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="setConditionsLabel">
                 <property name="text">
                  <string>Conditions</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="setConditionsAddButton">
                 <property name="text">
                  <string>+</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="setConditionsRemoveButton">
                 <property name="text">
                  <string>—</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableWidget" name="setConditionsTableWidget">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideNone</enum>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>32</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>20</number>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>20</number>
               </attribute>
               <column/>
               <column/>
               <column/>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="setAnimationsLabel">
               <property name="text">
                <string>Animations:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="setAnimationsAddButton">
               <property name="text">
                <string>+</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="setAnimationsRemoveButton">
               <property name="text">
                <string>—</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="setAnimationsTableWidget">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderMinimumSectionSize">
              <number>20</number>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>20</number>
             </attribute>
             <column/>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
